### DISCUSSION TITLE
Why removed table plugin from Extensions in github?

### DISCUSSION THREAD

**Discussion by senthil177cse** [GrapesJS/grapesjs#4436] [Q&A]
i want table plugin refrence code to implement my project. so can you enable that plugin in Extension?

**Comment by senthil177cse** [comment]
@artf  any uodate?

**Reply by artf** [reply]
Which table plugin are you talking about?

**Comment by ronaldohoch** [comment]
Hello!

I think you can create the blocks of table with a code similar to it: 
https://github.com/artf/grapesjs-preset-webpage/blob/master/src/blocks/index.js

```js
editor.BlockManager.add('text-basic', {
    category: 'Table',
    label: 'Table',
    attributes: { class: '' },
    content: `` // <- Your table html goes here.
  });
```

**Reply by senthil177cse** [reply]
already i have implemented thanks for the ans @ronaldohoch
